sklearn/utils/estimator_checks.py

```diff
@@ -123,6 +147,9 @@ def _yield_classifier_checks(name, classifier):
     yield check_classifiers_one_label
     yield check_classifiers_classes
     yield check_estimators_partial_fit_n_features
+    # test classifiers for multi-target output
```

Comment:
I don't find this comment more informative that the name of the function `check_classifier_multioutput`.
You might remove it.